src/lib/model/InboundTransfersModel.js

```diff
@@ -667,6 +668,18 @@ class InboundTransfersModel {
         }
     }
 
+    async sendNotificationToPayee(body, transferId) {
+        try {
+            const res = await this._backendRequests.putTransfersNotification(body, transferId);
+            return res;
+        } catch (err) {
+            this._logger.push({ err }).log('Error in getParticipants');
```

Comment:
copy/paste problem here @vgenev .... should be `Error in sendNotificationToPayee` I think.